src/historywork/WriteVerifiedCheckpointHashesWork.cpp

```diff
@@ -14,36 +15,74 @@
 #include "work/ConditionalWork.h"
 #include <Tracy.hpp>
 #include <algorithm>
+#include <filesystem>
 #include <fmt/format.h>
 
 namespace stellar
 {
+std::optional<LedgerNumHashPair>
+WriteVerifiedCheckpointHashesWork::loadLatestHashPairFromJsonOutput(
```

Comment:
This shouldn't return optional, since we just throw in every path that would return nullopt anyway